### ISSUE TITLE
Accessibility: Isotopic results prev & next anchor should have a role=button attribute

### ISSUE THREAD

**Issue by ernestmarcinko** [WPDreams/ajax-search-pro-development#75]
### Describe the bug

> An Improper Use of Link error appears if you have links that are missing an href attribute or are only linked to a #, and do not have role=”button” on them. Links should be used to direct people to other parts of your site. Any other functionality that triggers an action should be a button. To resolve this error you need to recode the link to use a <button> tag (preferable) or add role=”button” to the existing tag. If the element is a toggle button (such as an accordion), additional ARIA attributes are required.

[Reference support ticket](https://wp-dreams.com/forums/topic/accessibility-checker-suggests-imporper-use-of-link/)

- [x] Add the roles in the asp.shortcode.results.php file
